specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -4273,6 +4274,14 @@
             "modelAsString": true
           },
           "description": "Test coverage for the endpoint."
+        },
+        "locationDetails": {
+          "$ref": "#/definitions/ConnectionMonitorEndpointLocationDetails",
+          "description": "Location details for connection monitor endpoint."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "Subscription ID for connection monitor endpoint."
```

Comment:
>"Subscription ID for connection monitor endpoint." [](http://example.com/codeflow?start=25&length=50)

In description please explain it better when it is required or optional, and what exactly it corresponds to

Reply:
Hey, yes it's optional, only needed in case of ArcNetwork.